include/class_loader/class_loader_register_macro.h

```diff
@@ -46,7 +46,7 @@
     typedef  Base _base; \
     ProxyExec ## UniqueID() \
     { \
-      if (std::string(Message) != "") { \
+      if (not std::string(Message).empty()) { \
```

Comment:
Nitpick: As the entire library uses `!` for the negation operator and not the alternative operator `not`, I'd prefer to use `!` here

Reply:
Ok, done.